controller/node_controller.go

```diff
@@ -73,6 +75,17 @@ type NodeController struct {
 
 type TopologyLabelsChecker func(kubeClient clientset.Interface, vers string) (bool, error)
 
+// Ref: https://github.com/longhorn/longhorn/issues/7931
+type defectiveKernelRange struct {
+	broken string
+	fixed  string
+}
+
+var knownDefectiveKernels = []defectiveKernelRange{
+	{"5.15.0-94", "5.15.0-100"},
+	{"6.5.6", "6.5.7"},
+}
```

Comment:
I am wondering if we should dynamically fetch this list instead of hard-coding. We can use our upgrade responder server for this purpose. For airgap env, we can use a hard-coded list. WDYT?